app/src/main/java/io/apicurio/registry/auth/AppAuthenticationMechanism.java

```diff
@@ -80,12 +80,14 @@ public class AppAuthenticationMechanism implements HttpAuthenticationMechanism {
     @Info(category = "auth", description = "Enable basic auth", availableSince = "1.1.X-SNAPSHOT", registryAvailableSince = "3.X.X.Final", studioAvailableSince = "1.0.0")
     boolean basicAuthEnabled;
 
+    // TODO: Add suffix?
```

Comment:
We can also update these config properties from `apicurio.authn.basic-client-credentials.cache-expiration` to `apicurio.authn.basic-client-credentials.cache-expiration-mins` or `apicurio.authn.basic-client-credentials.cache-expiration-minutes`. It's not obvious at first glance what unit is it.

Reply:
Not sure we should rename config properties at this point, since 3.0 is already released upstream.  wdyt?

Reply:
That's fair, I was only thinking about GA... Well if anyone complains we can go through a proper deprecation process. I updated the docs at least.